test/integration/volumes_test.go

```diff
@@ -130,6 +130,30 @@ func TestWaitForVolumeLinodeIDInstance(t *testing.T) {
 	}
 }
 
+func TestUpdateVolume(t *testing.T) {
+	client, teardown := createTestClient(t, "fixtures/TestUpdateVolume")
```

Comment:
In order for fixtures to generate correctly, maybe setup a volume here:
```go
client, volume, teardown, err := setupVolume(t, "fixtures/TestUpdateVolume")
defer teardown()
```

Reply:
ooo yeah, that makes a lot more sense :) :+1: